modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx

```diff
@@ -365,6 +400,134 @@ export default function EditObjectField({
+				(settings) => settings.name === 'filters'
+			);
+
+			if (typeof filters?.value === 'string') {
+				const parsedFilterValue: ObjectFieldFilterSetting[] = JSON.parse(
+					filters.value
+				);
+
+				const parsedFilterJsonValue = parsedFilterValue.map(
+					(parsedFilterValue) => {
+						return {
+							...parsedFilterValue,
+							json: JSON.parse(parsedFilterValue.json as string),
+						};
+					}
+				);
+
+				const newFilter: ObjectFieldSetting = {
+					...filters,
+					value: parsedFilterJsonValue,
+				};
+
+				const newObjectFieldSettings:
+					| ObjectFieldSetting[]
+					| undefined = [
+					...objectFieldSettings.filter(
+						(setting) => setting.name !== 'filters'
+					),
+					newFilter,
+				];
+
+				setValues({
+					objectFieldSettings: newObjectFieldSettings,
+				});
+
+				const newAggregationFilters = parsedFilterJsonValue.map(
+					(parsedFilter) => {
+						const objectField = objectFields?.find(
+							(objectField) =>
+								objectField.name === parsedFilter.filterBy
+						);
+
+						if (objectField) {
+							const aggregationFilter: AggregationFilters = {
+								fieldLabel:
+									objectField.label[defaultLanguageId],
+								filterBy: parsedFilter.filterBy,
+								filterType: parsedFilter.filterType,
+								label: objectField.label,
+								objectFieldBusinessType:
+									objectField.businessType,
+								objectFieldName: objectField.name,
+								value:
+									objectField.businessType === 'Integer' ||
+									objectField.businessType === 'LongInteger'
+										? parsedFilter.json[
+												parsedFilter.filterType as string
+										  ]
+										: undefined,
+							};
+
+							if (
+								objectField.businessType === 'Date' &&
+								parsedFilter.filterType === 'range'
+							) {
+								const dateRangeFilterValues: ObjectFieldDateRangeFilterSettings =
+									parsedFilter.json[parsedFilter.filterType];
+
+								const aggregationFilterDateRangeValues: LabelValueObject[] = [
+									{
+										label: dateRangeFilterValues['ge'],
+										value: 'ge',
+									},
+									{
+										label: dateRangeFilterValues['le'],
+										value: 'le',
+									},
+								];
+
+								const dateRangeAggregationFilter: AggregationFilters = {
+									...aggregationFilter,
+									valueList: aggregationFilterDateRangeValues,
+								};
+
+								return dateRangeAggregationFilter;
+							}
+
+							if (objectField.businessType === 'Picklist') {
+								const picklistFilterValues: string[] =
+									parsedFilter.json[
+										parsedFilter.filterType as string
+									];
+
+								const picklistValueList: LabelValueObject[] = picklistFilterValues.map(
+									(picklistFilterValue) => {
+										return {
+											checked: true,
+											label: picklistFilterValue,
+											value: picklistFilterValue,
+										};
+									}
+								);
+
+								const picklistAggregationfilter: AggregationFilters = {
```

Comment:
This variable should be reamed:
```suggestion
								const picklistAggregationFilter: AggregationFilters = {
```